plc4j/integrations/apache-nifi/nifi-plc4x-processors/src/main/java/org/apache/plc4x/nifi/Plc4xSourceRecordProcessor.java

```diff
@@ -154,11 +163,24 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
 				}
 			});
 			PlcReadRequest readRequest = builder.build();
-			PlcReadResponse readResponse = readRequest.execute().get(10, TimeUnit.SECONDS);
+			final FlowFile originalFlowFile = fileToProcess;
 			resultSetFF = session.write(resultSetFF, out -> {
 				try {
```

Comment:
@chrisdutz :  is there any cleanup of the readRequest that should be done here before throwing out of this method?

Reply:
@chrisdutz still looking for your feedback here Chris